fvtest/compilertriltest/ILValidatorTest.cpp

```diff
@@ -46,6 +46,8 @@ INSTANTIATE_TEST_CASE_P(ILValidatorTest, IllformedTrees, ::testing::Values(
     "(method return=Address (block (areturn (aladd (aconst 4) (iconst 1)))))",
     "(method return=Address (block (areturn (aiadd (iconst 1) (aconst 4)))))",
     "(method return=Address (block (areturn (aladd (lconst 1) (aconst 4)))))",
+    "(method return=Address (block (ireturn (aiadd (luconst 1) (aconst 4)))))",
+    "(method return=Address (block (areturn (aladd (iuconst 1) (aconst 4)))))",
```

Comment:
`aladd` and `aiadd` are deprecated (see #2657) so these tests are not needed.

Reply:
I think you mean `luconst` right?